### DISCUSSION TITLE
Fancybox 4 filtering

### DISCUSSION THREAD

**Discussion by wtem-design** [fancyapps/ui#371] [Q&A] [answered]
Hi, how can I change what set of elements is selected to be shown in fancybox? 

I have a page with a list of elements that I filter, showing and hiding some of them. The elements can be shown in fancybox, and I want its content to show only filtered elements as well. I see that there was a selector option with earlier versions of fancybox, but is see no substitute for that in v4 docs.

I tried variations of this approach with no luck (meaning I always get the full set of elements inside fancybox).  
Simplified sandbox: https://codepen.io/baltazar-fajto/pen/PoazJEM

**Comment by fancyapps** [comment, marked as answer]
Hi,

1) Remove `data-fancybox="gallery"` from your links. Fancybox automatically attaches click event to any element having `data-fancybox` attribute;
2) You have to change your code so that is uses some class name instead of applying `style="display:none"`;
3) Then choose valid CSS selector that returns your links and use if for `Fancybox.bind()` method. Also, move it outside click handler.

**Reply by wtem-design** [reply]
Thank you!
